library_generation/test/generate_library_integration_test.sh

```diff
@@ -173,6 +173,11 @@ grep -v '^ *#' < "${proto_path_list}" | while IFS= read -r line; do
   echo "Generate library finished."
   echo "Compare generation result..."
   if [ $enable_postprocessing == "true" ]; then
```

Comment:
nit: `"${enable_postprocessing}"`

Reply:
thanks, I added this and a couple other format fixes